lib/learn_test/runner.rb

```diff
@@ -0,0 +1,65 @@
+module LearnTest
+  class Runner
+    attr_reader :repo, :options
+    SERVICE_URL = 'http://ironbroker.flatironschool.com'
+
+    def initialize(repo, options = {})
+      @repo = repo
+      @options = options
+      die if strategies.empty?
+    end
+
+    def run
+      strategies.each do |strategy|
+        strategy.check_dependencies
+        strategy.configure
+        strategy.run
```

Comment:
ah yeah that's a good point, i think i'll just take it back to running only one strategy